microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/keycloak/KeycloakIntegrationHighLevelScenarioTest.java

```diff
@@ -115,7 +115,12 @@ private static boolean isContainerReady(int port) {
             HttpURLConnection connection = (HttpURLConnection) url.openConnection();
             connection.setRequestMethod("GET");
 
-            return connection.getResponseCode() == HttpURLConnection.HTTP_OK;
+            boolean ready = connection.getResponseCode() == HttpURLConnection.HTTP_OK;
+            if (ready) {
```

Comment:
I'd like to be sure the code is reflecting the requirement you described in the PR description: here we actually want to wait for some more time _once_ (and even though) the health check has returned a HTTP 200, right?

Reply:
correct, even when server reports READY, the admin user might still not be created yet. That is also why it only makes sense to wait after healtch check HTTP 200